server/app.js

```diff
@@ -0,0 +1,64 @@
+const express = require('express');
+const bodyparser = require('body-parser');
+const db = require('./db/index.js');
+
+const app = express();
+
+app.set('port',3004);
+
+app.use(bodyparser.json());
+
+app.use(bodyparser.urlencoded({extended: true}));
+
+//app.use(express.static(__dirname + '' ));
+
+app.get('/api/alsovieweditems', (req, res) => {
+
+  var queryString = 'select * from alsovieweditems';
+  var queryArgs = [];
+
+  db.query(queryString, queryArgs, function(err, results) {
+    if (err) {
+      res.json(err);
```

Comment:
Is responding to the client with the actual error giving them too much info? Security concerns I believe.

Reply:
What error code you suggests for database error instead? I will also look into it.

Reply:
Maybe a 500 is good enough. Here is a cheat sheet with some explanation. https://www.cheatography.com/kstep/cheat-sheets/http-status-codes/